man/oci-runtime-tool-validate.1.md

```diff
@@ -15,8 +15,8 @@ Validate an OCI bundle
 **--help**
   Print usage statement
 
-**--path=PATH
-  Path to bundle
+**--path**=PATH
+  Path to bundle. The default is *.*.
```

Comment:
I think change this to `The default is current working directory.` maybe better.